src/algokit/core/deploy.py

```diff
@@ -0,0 +1,118 @@
+# 1. User can call algokit deploy to different networks
+# 2. By default algokit cli contains configs for testnet and mainnet
+# 3. User can overwrite them by creating a config file in the project root
+
+
+import logging
+import os
+import sys
+from pathlib import Path
+
+import click
+import httpx
+from dotenv import dotenv_values
+
+from algokit.core import proc
+from algokit.core.conf import get_algokit_config
+from algokit.core.constants import ALGOKIT_CONFIG, ALGORAND_NETWORKS, AlgorandNetworkConfiguration
+
+if sys.version_info >= (3, 11):
+    pass
+else:
+    pass
+
+logger = logging.getLogger(__name__)
+
+
+def get_genesis_network_name(deploy_config: dict) -> str | None:
+    """
+    Get the network name from the genesis block.
+    :param deploy_config: Deploy configuration.
+    :return: Network name.
+    """
+
+    algod_server = deploy_config.get("ALGOD_SERVER")
+
+    if not algod_server:
+        raise click.ClickException("Missing ALGOD_SERVER in deploy configuration.")
+
+    try:
+        genesis_response = httpx.get(f"{algod_server}/genesis")
+        genesis_response.raise_for_status()
+        return genesis_response.json()["network"] if genesis_response.status_code == 200 else None  # noqa: PLR2004
```

Comment:
you can avoid this linting error with using `httpx.codes.OK` instead of `200`

Reply:
done